core/management/commands/import_wordpress_pages.py

```diff
@@ -0,0 +1,69 @@
+import lxml.etree as etree
+from core.models import StaticPage
+from django.core.management.base import BaseCommand, CommandError
+
```

Comment:
Since this is a new script file perhaps it would be good to include comments at the top indicating its purpose and how it should be used?

Reply:
for sure, that's a good idea! this is a very "one off" script, but more details/a description would be good. I'll do that

Reply:
done in 1e7b57b